src/styles/utils/_functions.scss

```
@@ -0,0 +1,18 @@
+@for $date from 1 through 31 {
+  .calendar__cell:nth-child(#{$date})::before {
+    content: "#{$date}";
+  }
+}
+
+@for $day from 28 through 31 {
+  .calendar--month-length--#{$day}
+  :nth-child(n + #{$day + 1}) {
+    display: none;
+  }
+}
+
+@each $day, $distance in $start-day {
+  .calendar--start-day--#{$day} .calendar__cell:first-child {
+    margin-left: $distance;
+  }
+}
```

I'm not sure but it seems like it must works without this selector: .calendar__cell:first-child
```suggestion
@each $day, $distance in $start-day {
  .calendar--start-day--#{$day} {
    margin-left: $distance;
  }
}
```

---

it won’t if i removed it, pay attention to my code